### DISCUSSION TITLE
Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by Peoppon** [isontheline/pro.webssh.net#626] [Ideas]
Maybe add keyboard to allow for faster switching between terminal and SFTP tabs? And could be compatible with iPad (what I'm using) so could use option or command key (would then also be compatible with MacOS)? Could be similar to how safari deals with switching tabs (command + number to switch to that number tab, e.g. cmd + 1 switches to leftmost tab).

**Comment by isontheline** [comment]
Hello @Peoppon 👋

Very interesting request!

I'm opening an issue : https://github.com/isontheline/pro.webssh.net/issues/627
